data/translations/Internationalization_tr.ts

```diff
@@ -1535,7 +1535,7 @@ Lütfen bunları yapılandırma dosyasında elle çözün.</translation>
         <location filename="../../cmake-build-debug/src/flameshot_autogen/include/ui_infowindow.h" line="124"/>
         <location filename="../../cmake-build-relwithdebinfo/src/flameshot_autogen/include/ui_infowindow.h" line="124"/>
         <source>Copy Info</source>
-        <translation>Kopya Bilgisi</translation>
+        <translation>Bilgileri Kopyalai</translation>
```

Comment:
Typo, It should be "Bilgileri Kopyala" not "Bilgileri Kopyalai"

Reply:
Damn, missed that one, thank.

Reply:
This string also seems to be wrong. it should be changed to "Güncellemeleri otomatik denetle"

<details><summary>Screenshot</summary>

![annotely_image](https://github.com/flameshot-org/flameshot/assets/133198410/dbbc173b-f090-45d3-b3a2-609cdd1f1817)


</details>

Reply:
> This string also seems to be wrong. it should be changed to "Güncellemeleri otomatik denetle"
> Screenshot
> 
> ![annotely_image](https://user-images.githubusercontent.com/133198410/237597031-dbbc173b-f090-45d3-b3a2-609cdd1f1817.png)

Can confirm this